Resources/Prototypes/Recipes/Reactions/chemicals.yml

```diff
@@ -78,16 +76,16 @@
     PolytrinicAcid: 3
 
 - type: reaction
-  id: SulfuricAcid
-  reactants:
-    Hydrogen:
+  id: SulfuricAcid 
+  reactants: 
+    Sulfur: 
```

Comment:
Extra whitespace

Reply:
what do you mean? Elaborate please?

Reply:
You have extra spaces at the end of your lines.